gegelatilib/include/log/Logger.h

```diff
@@ -0,0 +1,58 @@
+#ifndef GEGELATI_LOGGER_H
+#define GEGELATI_LOGGER_H
+
+#include <iostream>
+
+namespace Log {
+
+/**
+* Logger class enabling to log elements in a given output stream.
+*/
+    class Logger {
+    private:
+        /**
+        * output stream where all what is logged is put.
+        */
+        std::ostream *out;
+
+    public:
+        /**
+        * \brief basic constructor logging on cout.
+        */
+        Logger() : out(&std::cout) {};
+
+        /**
+        * \brief constructor initializing a specific output.
+        * \param[in] out the output stream we want to log things to as ostream.
+        */
+        explicit Logger(std::ostream &out) : out(&out) {};
+
+        /**
+        * \brief << operator to manipulate stream and enter stream-specific
+        * elements (like std::endl).
+        * \param[in] manip the element we want to add.
```

Comment:
Put en empty line between the \brief and the rest of the comments. (to check everywhere else)